readme.md

```diff
@@ -1,16 +1,16 @@
-# Bulk Nominatim Plugin
+# LocationIQ Plugin
 
-The ***Bulk Nominatim QGIS Plugin*** interfaces with an OpenStreetMap nominatim service to provide geocoding of addresses and reverse geocoding of coordinates. This plugin is only recommended to be used with personal nominatim servers because OpenStreetMap does not permit bulk nominatim requests.
+The ***LocationIQ QGIS Plugin*** interfaces with our service to provide geocoding of addresses and reverse geocoding of coordinates. This plugin is forked from the **Bulk Nominatim QGIS Plugin** hosted at `https://github.com/NationalSecurityAgency/qgis-bulk-nominatim`
 
 ## Installation
-The plugin can be installed by downloading the latest version from `https://github.com/NationalSecurityAgency/qgis-bulk-nominatim`. Launch QGIS and select ***Plugins->Manage and Install Plugins...*** from the menu. Click on the ***Install from ZIP*** tab on the left, click on the `...` button on the right to navigate to where you saved the bulk nominatim downloaded ZIP file and select it then click on the ***Install Plugin*** button. Click the ***Close*** button and you are ready to start address geocoding.
+The plugin can be installed by downloading the latest version from `https://github.com/locationiq/locationiq-qgis`. Launch QGIS and select ***Plugins->Manage and Install Plugins...*** from the menu. Click on the ***Install from ZIP*** tab on the left, click on the `...` button on the right to navigate to where you saved the bulk nominatim downloaded ZIP file and select it then click on the ***Install Plugin*** button. Click the ***Close*** button and you are ready to start address geocoding.
 
 <div style="text-align:center"><img src="doc/install.jpg" alt="Installing Plugin"></div>
 
 
-## Bulk Nominatim Tools
+## LocationIQ Tools
 
-The plugin ***Settings*** need to be configured to point to a nominatim endpoint. See the ***Settings*** section on how to do this. The plugin can be accessed from the `Plugins->Nominatim GeoCoding` menu or from the two tool bar icons.
+The plugin ***Settings*** need to be configured to point to a nominatim endpoint. See the ***Settings*** section on how to do this. The plugin can be accessed from the `Plugins->LocationIQ GeoCoding` menu or from the two tool bar icons.
 
 <div style="text-align:center"><img src="doc/menu.jpg" alt="Nominatim GeoCoding"></div>
```

Comment:
This image needs to be updated

Reply:
The image was updated earlier. Did I miss something?